### ISSUE TITLE
future dates on commits

### ISSUE THREAD

**Issue by nkiesel** [seandepagnier/weather_routing_pi#46]
Some of your commits have dates that are in the future.  Current example is https://github.com/seandepagnier/weather_routing_pi/commit/23c07cc5d2be7ce68349f4b3719b6fa6fe90e0bf which has `Jun 19, 2152` as commit date.  This fools `git pull` which says my local repo is in sync with origin.

**Comment by nkiesel** [comment]
Another broken commit date is

`2106-02-07 06:28:56 -40643156 23c07cc ui updates and other changes`

**Comment by rgleason** [comment]
Norbert I beleve I git cloned this one and that was what I used last. Can you try a fresh git cline?
I certainly don't understand the dates.

**Comment by seandepagnier** [comment]
My chromebook is wel...

the usb ports are having serious issues.  It can't suspend at all.
50% of the keys are broken ( I must use usb keyboard..)  the speakers
blew up and melted inside, and the power button is broken.

What this means is when it is shut down, it won't turn on again unless
I take the screws out and manually reset it by unplugging the power
wires from the motherboard.  When I do this, the clock is usually more
than 100 years wrong.  I must have forgotten to manually type the
correct data as without internet I can't use ntp.

Is this a serious issue?  Should I try to correc it?  I am afraid it
would require everyone to clone the repository again from scratch.

Sean

**Comment by lanceberc** [comment]
I'll inform the head of Chromebook engineering when I see him next. Marine
environments were not his first design goal.

**Comment by rgleason** [comment]
Lance, Even better ask him if he has 3-4 chromebooks in overstock that he could contribute for some environmental testing! Might be a good story for them.

**Comment by nkiesel** [comment]
No, I would not worry about the timestamp.  It's really more a cosmetic issue. I would not even have mentioned it if I would have realized that I already had that commit.

It also helped to find a bug in Git itself (partial fix is queued and will be part of next Git release).  We were still not able to reproduce the exact scenario, especially the really large timezone offset from that commit.  Do you happen to know what the actual time and timezone is after such a hard reboot? Is this random or always the same (wrong) value? If the latter: can you please share it?
